### ISSUE TITLE
Incompatible with Apollo Federation

### ISSUE THREAD

**Issue by jasonav** [gabotechs/graphqxl#51]
When using Apollo Federation, you need this at the top of your schema file:

```
extend schema 
@link(url: "https://specs.apollo.dev/federation/v2.0", 
import: ["@key", "https://github.com/Shareable", "@inaccessible"])
```

GraphQXL balked at this. I worked around it by storing this in a different file and writing a small script to generate and then concatenate, but it would be better if GraphQXL just recognized this and left it alone.

**Comment by gabotechs** [comment]
I suppose that you are referring to something like this:
```graphql
extend schema 
@link(url: "https://specs.apollo.dev/federation/v2.0", 
import: ["@key", "https://github.com/Shareable", "@inaccessible"])
```
I think it's doable, I can have a PR prepared in some days for this

**Comment by jasonav** [comment]
Yes, sorry, the copy-paste munged it. Edited to fix.

**Comment by gabotechs** [comment]
This [PR](https://github.com/gabotechs/graphqxl/pull/52) that was just merged added support for the `extend` keyword as defined in the formal graphql specification, so you should be able to use it not only on the `schema` but also in `types`, `inputs`, `enums`, etc...

It was released under the `0.40.0` tag, which is currently available in npm (`npm install node-graphqxl`), and it will soon be available in the oficial `brew` repositories.

Let me know if you find any problems

**Comment by jasonav** [comment]
Works great. Thanks!
